django_wait_for_migrations.py

```diff
@@ -44,7 +44,7 @@ def wait_for_migrations(args):
         sleep(1)
         wait += 1
 
-        if wait > 30:
+        if wait > 500:
```

Comment:
Let's increase it to 2000 to see if it finally runs.

Reply:
🤔 didn't work either. I'm not sure if increasing it more will help

Reply:
Yeah, I'm not sure either. 🤔

Reply:
I'm going to try 4000, and then look at some other solution. Have you all ever ran into this issue before?